src/SFA.DAS.AssessorService.Application/Handlers/Certificates/UpdateCertificateHandler.cs

```diff
@@ -54,7 +56,10 @@ public async Task<Certificate> Handle(UpdateCertificateRequest request, Cancella
                 _logger.LogInformation($"Certificate with ID: {request.Certificate.Id} Updated with reference of {request.Certificate.CertificateReference}");
             }
 
-            updatedData = HandleSendToUpdate(currentCertificate, currentData, updatedData);
+            updatedData = CertificateDataSendToUpdater.HandleSendToUpdate(currentCertificate, currentData, updatedData);
+
+            updatedData.CoronationEmblem = await _standardRepository.GetCoronationEmblemForStandardReferenceAndVersion(updatedData.StandardReference, updatedData.Version);
+            updatedData.StandardName = await _standardRepository.GetTitleForStandardReferenceAndVersion(updatedData.StandardReference, updatedData.Version);
```

Comment:
This gets the CoronationEmblem and StandardName from the database every time the certificate is updated. This fulfills ACs 3.2 and 3.3, and I find it neater to have the Assessor decide this instead of having the Admin Service supply the correct values.

Reply:
There should be an equivalent class to the CertificateDataSentToUpdater i.e. CertficateDataVersionUpdater which checks if the version has changed and then does these updates as that is the pattern to follow and these update should only be done when the version has changed which the AC's do say

Reply:
Done. I've also fixed the existing tests in several places